_rules/SC1-2-1-video-only-description-track.md

```diff
@@ -0,0 +1,109 @@
+---
+name: Video only element description track
+description: |
+  This rule checks that description tracks that come with non-streaming `video` elements, without audio, are descriptive.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contain audio and a `track` element with a `kind="descriptions"` attribute.
+
+### Expectation
+
+The visual information of each test target is described with a description `track` element.
+
+*Note*: Multiple description `track` elements may be useful for different languages, but at least one must match the language of the video.
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```

Comment:
Please check the wording for this section in other media rules (#204)

Reply:
Done. Thank you.